package.json

```
@@ -60,8 +60,9 @@
     "arrowParens": "always"
   },
   "dependencies": {
+    "rc-tooltip": "5.1.0",
```

```suggestion
    "rc-tooltip": "^5.1.0",
```

Requiring an exact version prevents dependents from deduplicating if they don't also use exactly 5.1.0.

---

done